examples/README.md

```diff
@@ -0,0 +1,177 @@
+       auto out_tensor = ctx->AssociateValueAndTensor(n->outputs()[0], new_layer->getOutput(0));
+
+       LOG_DEBUG("Output shape: " << out_tensor->getDimensions());
+       return true;
+     }});
+
+} // namespace
+} // namespace impl
+} // namespace converters
+} // namespace conversion
+} // namespace core
+} // namespace trtorch
+```
+
+## Generate `.so` library
+To use this converter in Python, it is recommended to use PyTorch's
+[C++/CUDA Extension](https://pytorch.org/tutorials/advanced/cpp_extension.html#custom-c-and-cuda-extensions).
+We give an example here about how to wrap the converter into a `.so`
+library so that you can load it to use in Python applicaton. 
+```python
+import os
+from setuptools import setup, Extension
+from torch.utils import cpp_extension
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+# library_dirs should point to the libtrtorch.so, include_dirs should point to the dir that include the headers
+# 1) download the latest package from https://github.com/NVIDIA/TRTorch/releases/
+# 2) Extract the file from downloaded package, we will get the "trtorch" directory
+# 3) Set trtorch_path to that directory
+trtorch_path = os.path.abspath("trtorch")
+
+ext_modules = [
+    cpp_extension.CUDAExtension('elu_converter', ['elu_converter.cpp'],
+                                library_dirs=[(trtorch_path + "/lib/")],
+                                libraries=["trtorch"],
+                                include_dirs=[trtorch_path + "/include/trtorch/"])
+]
+
+setup(
+    name='elu_converter',
+    ext_modules=ext_modules,
+    cmdclass={'build_ext': cpp_extension.BuildExtension},
+)
+```
+Make sure to include the path for header files in `include_dirs` and the path 
+for dependent libraries in `library_dirs`. Generally speaking, you should download 
+the latest package from [here](https://github.com/NVIDIA/TRTorch/releases), extract
+the files, and the set the `trtorch_path` to it. You could also add other compilation 
+flags in cpp_extension if you need. Then, run above python scripts as:
+```shell
+python3 setup.py install --user
+```
+You should see the output similar to the contents indicated [here](https://pytorch.org/tutorials/advanced/cpp_extension.html#custom-c-and-cuda-extensions)   after running
+`python setup.py install`. You should find a couple of new folders generated
+by the command above. In build folder, you can find the generated `.so` library, 
+which could be loaded in our Python application. 
+
+## Load `.so` in Python Application
+With the new generated library, TRTorch now support the new developed converter. 
+We use `torch.ops.load_library` to load `.so`. For example, we could load the ELU 
+converter and use it in our application:
+```python
+import torch
+import trtorch
+
+# After "python3 setup install", you should find this .so file under generated "build" directory
+torch.ops.load_library('./build/lib.linux-x86_64-3.6/elu_converter.cpython-36m-x86_64-linux-gnu.so')

```

Comment:
I think you should also mention that you can also import the .so with `import elu_converter` (verify this works first)

Reply:
It seems that `import elu_converter` is not supported by this setting, it shows an error as: `ImportError: dynamic module does not define module export function (PyInit_elu_converter)`. I googled this bug, found that this happens often because of python2/3 linking problem. I tried to solve this by different methods but got no progress..

Reply:
No problem, its not a big deal